clients/src/main/java/org/apache/kafka/clients/admin/DeleteShareGroupsResult.java

```diff
@@ -20,13 +20,33 @@
 import org.apache.kafka.common.KafkaFuture;
 
 import java.util.Collection;
+import java.util.HashMap;
 import java.util.Map;
 
 /**
  * The result of the {@link Admin#deleteShareGroups(Collection <String>, DeleteShareGroupsOptions)} call.
  */
-public class DeleteShareGroupsResult extends DeleteGroupsResult {
+public class DeleteShareGroupsResult {
```

Comment:
Do we need to preserve the Evolving tag?

Reply:
That would be consistent with the rest of the KIP-932 interfaces, although not that critical in this case.

Reply:
@chia7712 had suggested to remove it saying the interface is mature.

https://github.com/apache/kafka/pull/19057#issuecomment-2698014961

Reply:
That seems wrong given that queues is still EA.

Reply:
Will rectify, thanks for pointing out.

Reply:
@chia7712 
@ijuma 
https://github.com/apache/kafka/pull/19133